database/migrations/2019_09_14_094017_update_gatwick_departures.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+use App\Models\Airfield;
+use App\Models\Sid;
+use Illuminate\Database\Migrations\Migration;
+
+class UpdateGatwickDepartures extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $gatwick = Airfield::where('code', 'EGKK')->firstOrFail()->id;
```

Comment:
Minor issue - what happens if `ModelNotFoundException` is thrown (I know it is a major edge-case and will only likely get run once, but I'd hate the deployment to fail)?

Reply:
I've sorta gone with the view that if I haven't accounted for all the required data, that we'll have bigger problems! That said, I've gone for the fail fast approach so we can quickly find what went wrong. Is there another, better way we could do this?

Reply:
I think this works actually.